VMEncryption/main/oscrypto/OSEncryptionState.py

```diff
@@ -201,9 +201,7 @@ def _add_kernelopts(self, args_to_add):
 
         grub_cfg_paths = filter(lambda path_pair: os.path.exists(path_pair[0]) and os.path.exists(path_pair[1]), grub_cfg_paths)
 
-        for grub_cfg_path, grub_env_path in grub_cfg_paths:
-            for arg in args_to_add:
-                self.command_executor.ExecuteInBash("grubby --args {0} --update-kernel ALL -c {1} --env={2}".format(arg, grub_cfg_path, grub_env_path))
+        self.context.distro_patcher.add_kernelopts(args_to_add, grub_cfg_paths)
```

Comment:
Can you move the the grub_cfg_paths logic to redhatPatching add_kernelopts function and only pass args_to_add parameter? It seems grub_cfg_path will change depending on the distro.

Reply:
Done!